### DISCUSSION TITLE
Add a column that shows all our likes

### DISCUSSION THREAD

**Discussion by Beageal** [robertodoering/harpy#872] [Ideas]
I'd love to be able to swipe over to a column that is only tweets that I've liked/favorited/♥️ed.

Example below from Fenix:
![Screenshot_20230115_005653_Fenix](https://user-images.githubusercontent.com/122721432/212532089-8255792b-9d5f-45b6-ac1b-65b6c038a60c.jpg)

(And if there was an option to display them in a random order, that'd be a delight.  But I'm probably the only person who'd get anything out of that...)

**Comment by robertodoering** [comment]
You view your likes on your profile, but I assume you'd like to add it to your home screen as well?
I've planned to be able to add user timelines into the home page, user likes would be nice to add as well

**Comment by Beageal** [comment]
OH I didn't even know there was a profile page like that!
Yes that's perfect!  Having it on the home screen would be great, but I'm totally fine with it as is.

Thanks for the info!  I've only had Harpy for a couple of days, but it's great!  Easy decision to upgrade to the paid version!
